### DISCUSSION TITLE
Just wait for more exploits

### DISCUSSION THREAD

**Discussion by The-Man-Is-Me** [3kh0/ext-remover#193] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
Don't go blowing up a chromebook or getting arrested. Just wait a month and it will probably be fixed. You don't need games that badly.

**Comment by 3kh0** [comment]
Just use my website 😎

**Reply by ghost** [reply]
plz giv ad revenue

**Reply by Theblackmamba824** [reply]
> Just use my website 😎

its blocked
the whole github.io is blocked

**Reply by 3kh0** [reply]
@Theblackmamba824 deploy your own copy of the site

**Reply by Theblackmamba824** [reply]
how

**Reply by ghost** [reply]
uh
if you need a copy, use mine. If it gets blocked, tell me. 
https://3kh0unblockedfork.netlify.app/

**Reply by ghost** [reply]
If you want to learn how to exploit, learn some basic js. The deploy instructions are on this repository, in the wiki.

**Reply by Theblackmamba824** [reply]
thx

**Reply by The-Man-Is-Me** [reply]
it blocks any renvition of it, al least for me

**Reply by Theblackmamba824** [reply]
datsss tufffff butttt cant u just go to the search bar and spell unblocked games google sites

**Reply by 3kh0** [reply]
just use my site tbh, 7kh0.github.io

**Reply by Theblackmamba824** [reply]
yh but for sum ppl its blocked
but i prefer your site
